### PR TITLE
Adds CI for build (any push / PR that is not master) and release (master)

### PR DESCRIPTION
**Pull request by jlengrand**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "reload": "open http://reload.extensions/",
     "start": "chokidar \"src/*.js\" -c \"npm run build;npm run reload\"\n\n",
     "build": "rimraf dist && rollup -c rollup.config.js",
-    "build:watch": "rimraf dist && rollup -c --watch rollup.config.js"
+    "build:watch": "rimraf dist && rollup -c --watch rollup.config.js",
+    "zip": "zip -r release/current.zip . -x \"node_modules/*\" \".idea/*\" \".git/*\" .gitignore",
+    "version": "mversion minor"
```
Comment (thepassle) [inline package.json:12]: does this also bump the manifest.json?

Comment (jlengrand) [reply package.json:12]: yeah that's why I use this and not npm version

Comment (jlengrand) [reply package.json:12]: Proof: https://github.com/open-wc/locator/pull/17/files#diff-4b1eb3dc48c4e16d49db5b42298fe654R5

**Comment by thepassle** [comment]
Thanks! ❤️

**Pull request merged**